File: logica-de-programacao/secao-10/exercicio/crescente.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){

    int n1, n2;

    while (n1 != n2){
        cout << "Digite dois numeros: " << endl;
        
        cin >> n1;
        cin >> n2;

        if (n1 < n2){
            cout << "CRESCENTE!!" << endl; 
        }
        else if (n2 < n1){
            cout << "DECRESCENTE!!" << endl; 
        }
    }
    
    return 0;
}

File: logica-de-programacao/secao-11/exercicios/crescente.java
package exercicios;

import java.util.*;

public class crescente {
    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        int n1, n2;

        n1 = 1;
        n2 = 2;

        while (n1 != n2) {
            System.out.println("Digite dois numeros: ");
            n1 = sc.nextInt();
            n2 = sc.nextInt();

            if (n1 < n2) {
                System.out.println("CRESCENTE!!");
            } else if (n2 < n1) {
                System.out.println("DECRESCENTE!!");
            }
        }

        sc.close();
    }
}
